src/Commands/CreateSites.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Rapidez\Statamic\Commands;
+
+use Statamic\Facades\Site;
+use Illuminate\Console\Command;
+use Rapidez\Core\Facades\Rapidez;
+
+class CreateSites extends Command
+{
+    protected $signature = 'rapidez:statamic:create:sites';
+
+    protected $description = 'Create sites based on the active Magento stores.';
+
+    public function handle(): int
+    {
+        $sites = [];
+        $stores = Rapidez::getStores();
+        $configModel = config('rapidez.models.config');
+
+        foreach ($stores as $store) {
+            Rapidez::setStore($store['store_id']);
+
+            $locale = $configModel::getCachedByPath('general/locale/code');
+            $lang = explode('_', $locale)[0] ?? '';
+            $url = $configModel::getCachedByPath('web/secure/base_url');
+
+            $sites[$store['code']] = [
+                'name' => $store['name'] ?? $store['code'],
+                'locale' => $locale,
+                'lang' => $lang,
+                'url' => $url,
+                'attributes' => [
+                    'magento_store_id' => $store['store_id'],
+                    'group' => $store['website_code'] ?? '',
+                    'disabled' => '{{ config:rapidez.statamic.sites.' . $store['code'] . '.attributes.disabled }}',
+                ]
+            ];
+        }
+
+        Site::setSites($sites);
+        Site::save();
```

Comment:
Since we're getting this from the database anyway, can't we maybe just overwrite the `getSavedSites` function in `Statamic\Sites\Sites` to just return a cached array with the queries already done? Then we set the sites on the fly and we can get rid of the sites.yaml all together 🤔

Reply:
Yes i agree, i feel like this might be the best way to go.
I cached the sites array for now because of the setStore() function which is needed to get the correct site data.
After https://github.com/rapidez/core/pull/567 gets merged we can use getValue() function without having to do the setStore().